### PR TITLE
Stimpacks

### PR DESCRIPTION
**Pull request by GoodWheatley**
## About the PR 

Adds the Stimpack from SS13. Once injected, it'll give a speed boost, faster stun recovery, ~~and small passive heal~~ for 5 minutes. Takes longer to purge from the body compared to other chems, thanks to the use of the metabolismRate modifier.

**Changelog**

:cl:
- add: Stimpacks are now available in uplinks for 8 TC.

### CODE REVIEW

**Comment by PixelTheKermit** [comment]
I feel like this will be a first purchase for most syndicate players

**Comment by moonheart08** [comment]
> I feel like this will be a first purchase for most syndicate players

just by running the damage numbers it looks like by the time it's all used up you'll be in crit on the floor unless you've got meds.

**Comment by moonheart08** [comment]
> > I feel like this will be a first purchase for most syndicate players
> 
> just by running the damage numbers it looks like by the time it's all used up you'll be in crit on the floor unless you've got meds.

wait no that's a /heal/, nvm this is definitely a must-purchase and i don't like that.

**Comment by GoodWheatley** [comment]
> > > I feel like this will be a first purchase for most syndicate players
> > 
> > 
> > just by running the damage numbers it looks like by the time it's all used up you'll be in crit on the floor unless you've got meds.
> 
> wait no that's a /heal/, nvm this is definitely a must-purchase and i don't like that.

I'd like it to have at least a lowgrade heal, but yeah 2 brute and burn per second is probably excessive.

**Comment by retequizzle** [comment]
2 brute/burn per second is overkill for something that also lowers stun duration and increases running speed. For reference, the heal rate on this is on par with omnizine just without toxin or airloss, while only taking one chem channel.

In addition, even if we kept these modifications as is, the price for this is... well, dirt cheap for what it offers. Price would need to be more than double to offset the utility of basically being uncatchable for 5 minutes outside of a CMO validhunting you with a hypospray while doped up on ephedrine themselves.

tl;dr - too good in current form, "must buy" items take variety off the table by proxy, either nerf heals or at least double the TC cost, 2 heal/sec is too high or 6 TC purchase price is too low, you pick.

**Comment by EmoGarbage404** [comment]
Honestly a single use 8 TC item isnt awful. Youre sacrificing a large amount of TC for something that'll immediately identify you as a traitor and target of the crew. Maybe knock off the burn heal and keep the rest as is.

**Comment by retequizzle** [comment]
> Honestly a single use 8 TC item isnt awful. Youre sacrificing a large amount of TC for something that'll immediately identify you as a traitor and target of the crew. Maybe knock off the burn heal and keep the rest as is.

Heal rate should still be halved with that in mind, disabler bolts only do 5 dmg per hit and the user has a guaranteed speed advantage for five minutes. Even without the speed advantage, you have a lower stun duration, so by the time they've caught up to you, you're already up and running again.

(to clarify, my pain points on this are too many objectively "good" perks with no real downside other than single use and the fact that you can buy multiple at the current pricing. 3 stimpacks is simply way too many for what this offers, even 2 is pushing it)

**Comment by PixelTheKermit** [comment]
Withdrawal when?

**Comment by GoodWheatley** [comment]
Tests have passed, ready for merge unless there's any balance changes needed

**Comment by mirrorcult** [comment]
ingame screenshots/video needed

**Comment by moonheart08** [comment]
Untested webedit, closing.
